### PR TITLE
Write README

### PR DESCRIPTION
**Pull request by edangelion**
## WHY

To get understanding of Usage
## WHAT

Write README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+$ pq2gorm                                                                                                                                   
+Usage: Generate gorm model structs from PostgreSQL database schema.
+  -d string
+    	Set output path (default "./")
+  -dir string
+    	Set output path (default "./")
+```
+
+Example 1: Generate gorm model files in current directory.
+```
```
Comment (dtan4) [inline README.md]: Please insert empty line above. See this about GitHub Flavoured Markdown: [Mastering Markdown · GitHub Guides](https://guides.github.com/features/mastering-markdown/)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,51 @@
...
+pq2gorm "postgresql://user:password@host:port/dbname?sslmode=disable"
+```
+
```
Comment (dtan4) [inline README.md:44]: Add sample output result here, like this:

---

For example, user model `user.go` as shown below will be generated:

``` go
type User struct {
    ID uint `json:"id"`
    ...
}
```

**Comment by edangelion** [comment]
Thank you.
I have fixed contents based on reviews.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,63 @@
...
+    	Set output path (default "./")
+```
+
+**Example 1:** Generate gorm model files in current directory.
+
+```bash
+$ pq2gorm "postgresql://user:password@host:port/dbname?sslmode=disable"
+```
+
+For an example, user model user.go as shown below will be generated:
```
Comment (dtan4) [inline README.md]: `For example` is correct.

**Comment by edangelion** [comment]
Sorry. I have fixed the sentence.

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**